pkg/scheduler/algorithm/predicates/csi_volume_predicate_test.go

```diff
@@ -23,13 +23,13 @@ import (
 	"testing"
 
 	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/storage/v1beta1"
```

Comment:
add an import alias `storagev1beta1` could be better as `pkg/scheduler/algorithm/predicates/predicates.go`

Reply:
Added alias